config/initializers/redis.rb

```diff
@@ -1,14 +0,0 @@
-# frozen_string_literal: true
-
-redis_connection = Redis.new(
-  url: ENV['REDIS_URL'],
-  driver: :hiredis
-)
-
-namespace = ENV.fetch('REDIS_NAMESPACE') { nil }
-
-if namespace
-  Redis.current = Redis::Namespace.new(namespace, redis: redis_connection)
-else
-  Redis.current = redis_connection
-end
```

Comment:
`config/initializers/stoplight.rb` is still using `Redis.current`, so this is likely to cause issues, at least when `REDIS_NAMESPACE`